File: LinearSearch/linear.py
def linearSearch(arry,target):
	for i in range(len(arry)):
		if arry[i]==target:
			return i
	else:
		return False

arry=[12,23,45,21,32,22,21,76,54,87,98]
values=int(input('Which number you have to Search?\n'))
print(linearSearch(arry,values))

File: LinearSearch/linear.js
function linearSearch(list,target){
    for(var i = 0; i < list.length; i++){
        if (list[i] === target) return list[i]; 
     
    }
    return -1;
}
const read=require("readline-sync");
input = read.question("enter your number.... \n");
var list=[12,23,45,21,32,22,21,76,54,87,98]
console.log(linearSearch(list,values))